common/types_test.go

```diff
@@ -325,11 +325,11 @@ func TestAddress_Scan(t *testing.T) {
 			}
 
 			if !tt.wantErr {
-				for i := range a {
-					if a[i] != tt.args.src.([]byte)[i] {
+				for i := range a.Bytes() {
```

Comment:
If this function were run multiple times, it would be better to save the result of a.Bytes() first into a variable and then use that variable.

Reply:
Corrected, this is not a problem on line 328. It evaluates it once and then computes range on the evaluation.

Still stands for 329 and 332 but not really an issue.